### ISSUE TITLE
Investigate Windows CI builds failing

### ISSUE THREAD

**Issue by GeckoEidechse** [R2NorthstarTools/FlightCore#280]
https://github.com/R2NorthstarTools/FlightCore/commit/675d75d817d4279b7a8630700d1feb691e4a7e31#commitcomment-109860468

**Comment by GeckoEidechse** [comment]
Actually I think the reason for failing builds is that Rust `1.69` was just released and our CI currently always pulls in newest stable.

When compiling on Windows locally with `1.68` it still compiles but throws a warning with

```
warning: the following packages contain code that will be rejected by a future version of Rust: nom v1.2.4, ntapi v0.3.7
```

which I guess is now rejected code ^^

**Comment by GeckoEidechse** [comment]
[`game-scanner`](https://crates.io/crates/game-scanner) seems to be the crate pulling in the outdated version of `ntapi`. Issue is it hasn't seen an update in a while so we might have to just drop it entirely.

**Comment by GeckoEidechse** [comment]
I [opened an issue with `game-scanner` upstream](https://github.com/EqualGames/game-scanner/issues/110) but depending on the time until response we might wanna consider dropping it.

It's only used for a single task atm which is getting Titanfall2 install location on Origin/EA-App and for that we could just write our own logic (which would also reduce number of dependencies used).

**Comment by GeckoEidechse** [comment]
Closed as completed by #281
